scripts/oauth_authorize.py

```diff
@@ -0,0 +1,308 @@
+#!/usr/bin/env python
+"""
+OAuth 2.0 Authorization Flow Helper for MCP Atlassian

```

Comment:
This helper script works well for handling user interaction. To improve user experience, you might consider adding an entry point like `mcp-atlassian --oauth-setup`. Then users don't need to clone this repo and run the script.

Reply:
good suggestion will do